### PR TITLE
DCC Ticketing Consent I (EXPOSUREAPP-10471)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
## Whats new
DCC Ticketing Consent I screen

Screenshot 2021-11-16 at 22 08 36

## Ticket
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10471)
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10194)

## How to test
1) Open Dev Menu
2) Scroll down to DCC Ticketing
3) Press "Open Consent I Screen" button
4) Check the consent screen

## ToDo
- [x] View model for this screen?

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml [hunk 1]
```diff
@@ -1,4 +1,44 @@
...
+    <!-- XTXT: DCC Ticketing: Consent One: Point 3 -->
+    <string name="dcc_ticketing_consent_one_point_3">"Die Ermittlung der geeigneten Zertifikate erfolgt lokal auf Ihrem Smartphone. Es werden dabei keine Daten an das RKI oder den Anbieter übermittelt."</string>
+    <!-- XTXT: DCC Ticketing: Consent One: Point 4 -->
+    <string name="dcc_ticketing_consent_one_point_4">"Das RKI erfährt die Daten zu Ihrer Buchung oder den von Ihnen gespeicherten Zertifikaten nicht."</string>
+    <!-- XTXT: DCC Ticketing: Consent One: Data protection -->
+    <string name="dcc_ticketing_consent_one_data_protection">"Ausführliche Hinweise zur Datenverarbeitung finden Sie in der Datenschutzerklärung"</string>
+    <!-- XBUT: DCC Ticketing: Consent One: Consent Button -->
+    <string name="dcc_ticketing_consent_one_consent_btn">"Einverstanden"</string>
+    <!-- XBUT: DCC Ticketing: Consent One: Cancel Button -->
+    <string name="dcc_ticketing_consent_one_nope_btn">"Nein Danke"</string>
```
Comment (KathrinLuecke) [inline Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml:33]: Please change "Nein Danke" to "Abbrechen" to make it consistent with iOS.
I will also correct this in Figma.

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]
Please change dcc_ticketing_consent_one_nope_btn

**Review by SamuraiKek — COMMENTED** [review]
Looks good, only thing I've noticed is the dialog that shows up when clicking  "Abbrechen" has the buttons switched compared to figma.

Screenshot_20211117-162712_CWA TEST

**Review by SamuraiKek — APPROVED** [review]
Retested. LGTM.

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by mtwalli — APPROVED** [review]

**Pull request merged**